src/Makefile

```diff
@@ -118,6 +118,11 @@ uninstall:
 	test ! -f $(BINDIR)/atdgen.exe || rm $(BINDIR)/atdgen.exe
 	ocamlfind remove atdgen
 
+findlib-install:
+	ocamlfind install atdgen META \
+		 $(MLI) $(CMI) $(CMO) $(CMX) $(CMXS) $(O) \
+			atdgen.cma atdgen.a atdgen.cmxa
+
```

Comment:
Should `install` still `ocamlfind install atdgen`?? Also, `findlib-install` should be `.PHONY`?